src/query/datavalues/src/types/deserializations/timestamp.rs

```diff
@@ -53,7 +53,6 @@ impl TypeDeserializer for TimestampDeserializer {
         for row in 0..rows {
             let mut reader = &reader[step * row..];
             let value: i64 = reader.read_scalar()?;
-            check_timestamp(value)?;
```

Comment:
don't need to check the timestamp?

Reply:
It's de from binary mode (we ensure it's right we serialize) , we can skip this check to improve the performance.

Reply:
ok